changelog.d/3476.changed

```diff
@@ -0,0 +1,5 @@
+os_version must be in lowercase
+path for winpe.wim in case of using wimboot
+BCD: Windows Boot Application (13200003) -> Windows Boot Loader \windows\system32\winload.exe
+BCD: Timeout: 30 -> 0
+Windows templates
```

Comment:
Let's please try to summary this. There will be a link to the issue/PR and as such the people can always read the detailed commit messages then.